src/Modules/Content/Dnn.PersonaBar.Pages/Pages.Web/src/components/PageTypeSelector/PageTypeSelector.jsx

```diff
@@ -51,13 +77,13 @@ class PageTypeSelector extends Component {
                                 {createdDate}
                             </span>
                         </div>
-                        <div className="page-info-item">
-                            <span className="page-info-item-label">
+                        <div className="page-info-item" style={pageParentInfoStyle}>
```

Comment:
I'd rather we avoid inline styles and instead use class or attribute selectors. Is there some problem with selector specificity which blocks us?

What I have in mind is to set a class based on the logic above

```
<div className="page-info-item {hierarchy.length < 31 ? page-info-long : null}" />
```

And style those divs purely in css. Thoughts @emac80 ?